package.json

```diff
@@ -49,8 +49,9 @@
     "@babel/preset-react": "7.27.1",
     "@commitlint/cli": "19.8.0",
     "@commitlint/config-conventional": "18.4.4",
-    "@contentful/f36-components": "4.70.0",
-    "@contentful/f36-tokens": "4.2.0",
+    "@contentful/f36-components": "5.1.0",
+    "@contentful/f36-tokens": "5.1.0",
+    "@phosphor-icons/react": "2.1.8",
```

Comment:
Why the extra dependency here? the phosphor icons should come via the package itself

Reply:
Yes, but we had a [cjs/esm](https://contentful.slack.com/archives/C961BF4RX/p1756811802159099) issue with Phosphor that [this](https://github.com/contentful/forma-36/pull/3170) should fix, when its merged. So I'll clean this.